examples/nek5000/make-nek-examples.sh

```diff
@@ -49,7 +49,13 @@ if [[ "$#" -eq 1 && "$1" -eq "clean" ]]; then
   exit 0
 fi
 
-: ${NEK5K_DIR:=`cd "../../../Nek5000"; pwd`}
+# Set defaults for the parameters
+if [ ! -f $NEK5K_DIR/bin/makenek ]; then
+  NEK5K_DIR:=`cd "../../../Nek5000"; pwd`
+fi
+
+# : ${NEK5K_DIR:=`cd "../../../Nek5000"; pwd`}
```

Comment:
If you're going to replace code, please actually replace it rather than leaving the old thing sitting around commented. In this case, I think this commit should be amended to go back to the old version.

Reply:
Several people have made changes to this code, and I wanted to give @thilinarmtb the opportunity to clean it up in the way he prefers.